nusantara/nusa_datasets/identic/identic.py

```diff
@@ -0,0 +1,392 @@
+# coding=utf-8
+# Copyright 2022 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""\
+Data loader implementation for IDENTICv1.0 dataset.
+"""
+
+import csv
+from pathlib import Path
+from typing import Dict, List, Tuple
+
+import datasets
+import pandas as pd
+
+from nusantara.utils import schemas
+from nusantara.utils.common_parser import load_ud_data
+from nusantara.utils.configs import NusantaraConfig
+from nusantara.utils.constants import Tasks
+
+_CITATION = """\
+@inproceedings{larasati-2012-identic,
+    title = "{IDENTIC} Corpus: Morphologically Enriched {I}ndonesian-{E}nglish Parallel Corpus",
+    author = "Larasati, Septina Dian",
+    booktitle = "Proceedings of the Eighth International Conference on Language Resources and Evaluation ({LREC}'12)",
+    month = may,
+    year = "2012",
+    address = "Istanbul, Turkey",
+    publisher = "European Language Resources Association (ELRA)",
+    url = "http://www.lrec-conf.org/proceedings/lrec2012/pdf/644_Paper.pdf",
+    pages = "902--906",
+    abstract = "This paper describes the creation process of an Indonesian-English parallel corpus (IDENTIC).
+    The corpus contains 45,000 sentences collected from different sources in different genres.
+    Several manual text preprocessing tasks, such as alignment and spelling correction, are applied to the corpus
+    to assure its quality. We also apply language specific text processing such as tokenization on both sides and
+    clitic normalization on the Indonesian side. The corpus is available in two different formats: plain',
+    stored in text format and morphologically enriched', stored in CoNLL format. Some parts of the corpus are
+    publicly available at the IDENTIC homepage.",
+}
+"""
+
+_DATASETNAME = "identic"
+
+_DESCRIPTION = """\
+IDENTIC is an Indonesian-English parallel corpus for research purposes.
+The corpus is a bilingual corpus paired with English. The aim of this work is to build and provide
+researchers a proper Indonesian-English textual data set and also to promote research in this language pair.
+The corpus contains texts coming from different sources with different genres.
+"""
```

Comment:
I think the tagset follows https://septinalarasati.com/morphind/#tagset, perhaps we can add this information on the `_DESCRIPTION`, something like: the tagset follows [MorphInd tagset](https://septinalarasati.com/morphind/#tagset) (Larasati et. al., 2011).

Reply:
And perhaps, since there is POS tagging, the description needs to be updated as well